dolphinscheduler-dist/release-docs/licenses/ui-next-licenses/LICENSE-eslint

```diff
@@ -0,0 +1,19 @@
+Copyright OpenJS Foundation and other contributors, <www.openjsf.org>
```

Comment:
Dependencies in `devDependencies` don't need to be included in the release licenses

Reply:
This piece may be adjusted in the future, so I wrote all the dependencies.